lab-andrew/package.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
This is a good start but your description tag is not properly configured, it should just be an informative sentence or two.  Additionally, you may wish to specify the "scripts">"test" to be "mocha" and add it to your dev-dependencies.